src/api/endpoints/logged_activities/approve.py

```diff
@@ -45,10 +45,12 @@ def put(self, logged_activity_id=None):
 
             request_approval_data = bulk_approval_query.all()
 
-            request_approval =bulk_approval_query.update({'status': 'approved'},
+            request_approval = bulk_approval_query.update({'status': 'approved'},
             synchronize_session=False)
 
             if request_approval:
+                for approval in request_approval_data:
+                    approval.society._total_points = approval.society._total_points + approval.value
```

Comment:
```python
_total_points
```
is a private variable, that should not be accessed directly, use `total_points` setter.

Reply:
`total_points()` doesn't work. It returns the value of`_total_points` when called and that I can't work with. Why should `_total_points` be private.

Reply:
> ```python
> _total_points
> ```
> 
> is a private variable, that should not be accessed directly, use `total_points` setter.

I get you. Fixed. Thanks.